docs/developers/frontend-shared.rst

```diff
@@ -0,0 +1,67 @@
+=========================
+Frontend Shared Package
+=========================
+
+The frontend-shared package is located in a sub directory called ``./frontend-shared`` inside this
+repository. The purpose of this package is to serve as a common library for shared utilities,
+components and styles that span across our frontend applications.
+
+The long term goal is to move this package into its own repository, but in the short term it
+lives in the client as a convenience to leverage the existing tooling. This also gives developers
+the benefit of working with just one repository when making changes that mostly affect the client
+in the first place.
+
+How it works
+=========================
+
+Frontend-shared works by creating a symlink via ``yarn link`` between the working dir
+``./frontend-shared/`` and the installed dir ``node_modules/@hypothesis/frontend-shared/``.
+
+Client setup
+=========================
+
+To setup frontend-shared in the client, simply run
+
+``$ make dev``
```

Comment:
I presume you mean that one should run `make dev` as one normally would for the entire client?

```suggestion
To set up ``frontend-shared`` in the client, run

``$ make dev``

as normal for development in the ``client`` application.
```

(Note that _set up_ in verb form is two words.)
(And I will always smoosh any use of the word _simply_) :)